dts/bindings/adc/renesas,ra-adc.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: Renesas ra ADC node
```

Comment:
```suggestion
description: Renesas RA ADC node
```

Reply:
We updated. Thank you for your feedback.